src/com/sidebar/guides/guides.js

```diff
@@ -0,0 +1,30 @@
+import {h, Component}	from 'preact/preact';
+import SVGIcon			from 'com/svg-icon/icon';
+
+export default class SidebarGuides extends Component {
+	constructor() {
```

Comment:
This constructor should probably have `super(props);`
 in or not be there. Although I'm pretty sure it doesn't make an actual difference.